cluster-scope/overlays/nerc-ocp-edu/clusterrolebindings/gpu-class-clusterroles.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kueue-clusterqueue-reader
+rules:
+  - apiGroups: ["kueue.x-k8s.io"]
+    resources: ["clusterqueues"]
+    verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: node-reader
```

Comment:
We already have a clusterrole called `nerc-node-reader` which could be used instead of adding a new/duplicate one.